pkg/worker/common.go

```diff
@@ -219,19 +107,20 @@ func (wp *persistentCatalogFileToEmbWorkerPool) saveChunks(
 	}
 
 	saveToMinIO := func(chunkUIDs []string) (map[string]string, error) {
-		chunksForMinIO := make(map[minio.ChunkUIDType]minio.ChunkContentType, len(textChunks))
+		chunksForMinIO := make(map[string][]byte, len(textChunks))
 		for i, uid := range chunkUIDs {
-			chunksForMinIO[minio.ChunkUIDType(uid)] = minio.ChunkContentType([]byte(texts[i]))
+			chunksForMinIO[uid] = []byte(texts[i])
 		}
 
-		if err := wp.svc.MinIO().DeleteTextChunksByFileUID(ctx, kbUID, kbFileUID); err != nil {
```

Comment:
I see we removed the chunk, embedding and converted file deletion as the previous step from saving them. However, I don't see where this old data cleanup is done when reprocessing a file. We just fixed this in https://github.com/instill-ai/artifact-backend/pull/259, but it seems we're back at increasing our blob and vector storage every time we reprocess a file.

Reply:
Addressed in the [commit](https://github.com/instill-ai/artifact-backend/pull/264/commits/b05daadb715fe7419d526ead4612d47289b6b901) with integration test for reprocessing file logic.